src/appleseed/renderer/modeling/environmentedf/latlongmapenvironmentedf.cpp

```diff
@@ -431,12 +440,13 @@ namespace
             assert(v >= 0.0f && v < 1.0f);
 
             InputValues values;
+            m_inputs.evaluate_uniforms(&values);
```

Comment:
I believe calling `evaluate_uniforms()` is unnecessary, `evaluate()` evaluates both uniform and varying inputs.